### PR TITLE
[SectionHeader] Add new component

### PR DESCRIPTION
**Pull request by ricardomv**
Used to collapse items in a list view.
Change demo to use new component in the icons list

http://www.google.com/design/spec/components/lists-controls.html#

### CODE REVIEW

File: modules/Material/ListItems/SectionHeader.qml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/*
+ * QML Air - A lightweight and mostly flat UI widget collection for QML
```
Comment (jneidlinger) [inline modules/Material/ListItems/SectionHeader.qml]: Comment header is outdated

Comment (ricardomv) [inline modules/Material/ListItems/SectionHeader.qml]: whoops, fixed

**Comment by ricardomv** [comment]
Added main icon ^^

**Comment by iBeliever** [comment]
Great work, thanks! :+1:

**Pull request closed without merging**